microsoft/PowerToys
# Get rid of indexes in dropdowns

## Summary of the Pull Request
It is refactoring of UI of KBM.

_What is this about?_
Right now to programmatically set something in a dropdown by keycode we have to find the index of keycode in the corresponding array and then call `SelectedIndex` method. The same logic is true for getting keycode from a dropdown. Here we remove unnecessary index layer.
## PR Checklist
* [X] Applies to #6913

## Info on Pull Request
We set a keycode as `DataContext` for `ComboBoxItem` and associate it with the value of `ComboBox`. 
If nothing is selected in a dropdown we say that keycode is `-1` similar to the previous implementation where we say that the selected index is `-1`. In that way, we do not have much to change in our code.

_What does this include?_

## Validation Steps Performed
* Run unit tests
* Manual test

_How does someone test & validate?_
* Do manual check in order to test for regressions
